gateway/src/apicast/policy/logging/readme.md

```diff
@@ -1,13 +1,13 @@
 # Logging policy
 
-This policy has two primary purposes: one is to enable and disable access log
-output, and the second one is to be able to create a custom access log format
-for each service and be able to set conditions to write custom access log. 
+This policy has two purposes: one is to enable and disable access log output,
+and and the other is to create a custom access log format for each service and
```

Comment:
`and the other is to create a custom access log format for each service, as well as to` 

Remove the first `and`.